src/components/modal/Modal.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Modal as RSModal} from 'reactstrap';
+import classnames from 'classnames';
+
+class Modal extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      modalOpen: props.is_open
+    };
+  }
+
+  toggle() {
+    if (this.props.setProps) {
+      this.props.setProps({is_open: !this.state.modalOpen});
+    } else {
+      this.setState({modalOpen: !this.state.modalOpen});
+    }
+  }
+
+  static getDerivedStateFromProps(nextProps, prevState) {
+    if (nextProps.is_open != prevState.modalOpen) {
+      return {modalOpen: nextProps.is_open};
+    } else return null;
+  }
+
+  render() {
+    const {children, className, scrollable, ...otherProps} = this.props;
+    const classes = classnames(className, {
+      'modal-dialog-scrollable': scrollable
+    });
```

Comment:
This is a temporary work-around, a `scrollable` prop for `Modal` will make its way into a future release of reactstrap, see [here](https://github.com/reactstrap/reactstrap/pull/1435). When it does, this should be removed in favour of passing the `scrollable` prop down to reactstrap.